spring-rabbit/src/main/java/org/springframework/amqp/rabbit/config/DirectRabbitListenerContainerFactory.java

```diff
@@ -104,12 +105,10 @@ protected DirectMessageListenerContainer createContainerInstance() {
 	protected void initializeContainer(DirectMessageListenerContainer instance, RabbitListenerEndpoint endpoint) {
 		super.initializeContainer(instance, endpoint);
 
-		if (this.taskScheduler != null) {
-			instance.setTaskScheduler(this.taskScheduler);
-		}
-		if (this.monitorInterval != null) {
-			instance.setMonitorInterval(this.monitorInterval);
-		}
+		JavaUtils javaUtils = JavaUtils.INSTANCE.acceptIfNotNull(this.taskScheduler, instance::setTaskScheduler)
```

Comment:
I wonder if we need to promote such an approach to Spring Framework directly.
Or if they have already something on the matter...

I have similar method complexity problem in Spring Integration and some common utility would save me.